maker/src/connection.rs

```diff
@@ -94,16 +94,18 @@ pub struct TakerMessage {
     pub msg: wire::MakerToTaker,
 }
 
+pub struct TakerMessageIgnoreErr(pub TakerMessage);
```

Comment:
Please add a comment describing the reason for wrapping this type

Reply:
Ah, I remember that this is something that we discussed - the alternative to having this type and having its own handler would be to replace the sink somewhere else with just an address send which intentionally swallows the error. The reason for this now is that MessageSink only exists for `Handler<Return = ()>`, to avoid swallowing errors or return values accidentally.

Reply:
if this is preserving the old behaviour, and just forcing us to be explicit about dropping the errors, this is also fine!

Reply:
The other change with sinks was about backpressure - now, it will wait for the messages it sends in to be handed before it sends the next.

Reply:
I'll reimplement this with `sink::unfold` to be equivalent to the old version - throwing away errors and no backpressure. It should be pretty easy to change to logging errors or stopping the sink on errors with an `unfold`-based sink